drivers/gpu/drm/panel/Kconfig

```
@@ -812,6 +812,14 @@ config DRM_PANEL_STARTEK_KD070FHFID015
 	  with a resolution of 1024 x 600 pixels. It provides a MIPI DSI interface to
 	  the host, a built-in LED backlight and touch controller.
 
+config DRM_PANEL_TD4320_BOEPLUS
```

```suggestion
config DRM_PANEL_BOE_TD4320
```
All other BOE drivers use pattern DRM_PANEL_BOE_... :thinking: